Libplanet.RocksDBStore/RocksDBKeyValueStore.cs

```diff
@@ -36,6 +36,18 @@ public void Set(byte[] key, byte[] value)
             _keyValueDb.Put(key, value);
         }
 
+        public void Set(IDictionary<byte[], byte[]> values)
```

Comment:
Could we have a dedicated test case for this, without indirection through other methods?

Reply:
Amended. https://github.com/planetarium/libplanet/pull/1570/commits/7a047b6c6ed40432caefb3018e4b2cc2e1a2ec9f